doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -59,7 +59,7 @@ org.kie.server.controller=http://<HOST>:<PORT>/controller/rest/controller
 ----
 +
 `org.kie.server.router.config.watcher.enabled` is an optional settings to enable the watcher service system property.
-
+. On every {KIE_SERVER} instance that must connect to the Smart Router, set the `org.kie.server.router` system property to the Smart Router URL.
```

Comment:
```suggestion
. On every {KIE_SERVER} instance that connects to the Smart Router, set the `org.kie.server.router` system property to the Smart Router URL.
```

Reply:
I don't think I can go with this as the instance does not actually connect to the Smart Router before this setting.